docs/index.md

```diff
@@ -20,13 +20,13 @@
 
 ## Compatibility
 
-pyjelly is compatible with Python 3.9 and newer. Python 3.9–3.13 are tested in CI and are guaranteed to work.
+pyjelly is compatible with Python 3.10 and newer. Python 3.10–3.14 are tested in CI and are guaranteed to work.
 
 The following table shows the compatibility of the pyjelly implementation with other libraries:
 
 | pyjelly                                                                                                                                             | Python     | RDFLib         |
 |-----------------------------------------------------------------------------------------------------------------------------------------------------|------------|----------------|
-| [0.1.0](https://w3id.org/jelly/pyjelly/0.1.0) – **[{{ python_package_version() }}](https://w3id.org/jelly/pyjelly/{{ python_package_version() }})** | 3.9 – 3.13 | 7.x (>= 7.1.4) |
+| [0.1.0](https://w3id.org/jelly/pyjelly/0.1.0) – **[{{ python_package_version() }}](https://w3id.org/jelly/pyjelly/{{ python_package_version() }})** | 3.10 – 3.14 | 7.x (>= 7.1.4) |
```

Comment:
nooooo

Please add a new row starting with 0.7.0 where python version support changed.

See how it's done in Jelly-JVM docs if in doubt.